cmd/cleanup/runner.go

```diff
@@ -207,6 +208,31 @@ func (r *runner) run(ctx context.Context, _ *cobra.Command, _ []string) error {
 		}
 	}
 	r.logger.LogCtx(ctx, "message", "deleted release CR")
+
+	r.logger.LogCtx(ctx, "message", fmt.Sprintf("deleting namespace %#q", r.flag.ClusterID))
```

Comment:
```suggestion
	r.logger.LogCtx(ctx, "message", fmt.Sprintf("waiting for %#q namespace deletion", r.flag.ClusterID))
```

@tomahawk28 @MarcelMue What about this? It's basically the same case as kvmconfig deletion.

Reply:
Not sure - I made my suggestions looking at the `Release CR` deletion :P 

No strong opinions

Reply:
I'm okay with it when Marcell is okay :trollface: